doc/sphinx/source/quickstart/installation.rst

```diff
@@ -503,14 +503,20 @@ code (called ``ESMValTool`` if you did not choose a different name)
 
     cd ESMValTool
 
-and run (on Linux only!)
+and (when on Linux) create a new environment called ``esmvaltool``
+containing just Python with the command
 
 .. code-block:: bash
 
-    conda env create --name esmvaltool --file environment.yml
+    conda create --name esmvaltool 'python=3.9'
+
+if needed, older versions of Python can also be selected.
```

Comment:
maybe add a note that env will might not solve for older versions (the GA still works but not sure how long til it'll break)

Reply:
Let's address the problem when it arises

Reply:
okay